web/src/main/java/org/springframework/security/web/FilterChainProxy.java

```diff
@@ -190,29 +198,33 @@ public void doFilter(ServletRequest request, ServletResponse response,
 	private void doFilterInternal(ServletRequest request, ServletResponse response,
```

Comment:
Rather than having this entire method be between a try/catch, can you please move the try/catch to the existing try block in `doFilter`?